modules/core/include/opencv2/core/hal/interface.h

```diff
@@ -78,6 +78,11 @@ typedef signed char schar;
 #define CV_32F  5
 #define CV_64F  6
 #define CV_16F  7
+#define CV_64S  8
+#define CV_64U  9
+#define CV_BOOL 10
```

Comment:
The `CV_BOOL` conflicts with the `CV_BOOL` of `modules/gapi/include/opencv2/gapi/gcommon.hpp`

Reply:
temporary solution: `CV_BOOL` -> `CV_1BOOL`.